finesse/hardware/plugins/stepper_motor/st10_controller.py

```diff
@@ -219,7 +219,7 @@ def close(self) -> None:
 
     @Slot()
     def _send_move_end_message(self) -> None:
-        self.send_message("move.end")
+        self.send_message("move.end", moved_to=self.angle)
```

Comment:
How about adding a `send_move_end_message` method to the `StepperMotorBase` class? I know it's only a one-liner, but it seems a bit tidier. Then we could get rid of `ST10Controller`'s `_send_move_end_message` method altogether too.

Reply:
Is [this](2ef80f771b94889527647829c85ddbe0d1c5f9c4) what you had in mind? Wasn't sure what to do with the `Slot` decorator, but if so I'll try and get a bit more test coverage and then re-request review.

Reply:
Yes, exactly! I'm still not entirely clear on what `Slot`s are for, but it seems like you can omit the decorator without problems, so I think it's fine.